src/utils/types.js

```diff
@@ -116,4 +116,22 @@ export const createSourceWithParamsPropType = {
   postalCode: PropTypes.string,
   country: PropTypes.string,
   card: PropTypes.string,
+  number: PropTypes.string,
```

Comment:
why card params described on the top level?

Reply:
I noticed that there were already other parameters related to the payment method, such as 'iban' or 'bank'